.idea/runConfigurations/instance_provision.xml

```diff
@@ -0,0 +1,21 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Why you adding those intellij files?

Reply:
![image](https://user-images.githubusercontent.com/10673561/65011614-6777f980-d914-11e9-9479-008acfc8129a.png)

for having such nice integration in IntelliJ

no longer need to write `sh gradlew` ... :)